src/PrestaShopBundle/Form/Admin/Sell/Product/ProductPriceType.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShopBundle\Form\Admin\Sell\Product;
+
+use PrestaShop\PrestaShop\Core\Form\FormChoiceProviderInterface;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\MoneyType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+class ProductPriceType extends AbstractType
+{
+    /**
+     * @var FormChoiceProviderInterface
+     */
+    private $taxRuleGroupChoiceProvider;
+
+    public function __construct(FormChoiceProviderInterface $taxRuleGroupChoiceProvider)
+    {
+        $this->taxRuleGroupChoiceProvider = $taxRuleGroupChoiceProvider;
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('price_tax_excluded', MoneyType::class, [
+                'divisor' => 6,
```

Comment:
```suggestion
                'divisor' => static::PRICE_DIVISOR,
```

Reply:
Where do you think we should define it? I guess it can be used in whole PS, so defining it in form type is not correct. :thinking: